test/gui/shared/scripts/helpers/api/Provisioning.py

```diff
@@ -58,3 +59,35 @@ def add_user_to_group(user, group_name):
         ocis.add_user_to_group(user, group_name)
     else:
         oc.add_user_to_group(user, group_name)
+
+
+def create_user(username):
+    user = {}
+    if username in helpers.UserHelper.test_users:
+        user = helpers.UserHelper.test_users[username]
```

Comment:
can we use it like this? if not then `import helpers.UserHelper as UserHelper`
```suggestion
    if username in UserHelper.test_users:
        user = UserHelper.test_users[username]
```